code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -191,142 +168,20 @@ Difficulty: Hard
-	if(handedness)
-		new /obj/effect/temp_visual/bubblegum_hands/rightsmack(T)
-	else
-		new /obj/effect/temp_visual/bubblegum_hands/leftsmack(T)
-	sleep(2.5)
-	for(var/mob/living/L in T)
-		if(!faction_check_mob(L))
-			to_chat(L, "<span class='userdanger'>[src] rends you!</span>")
-			playsound(T, attack_sound, 100, 1, -1)
-			var/limb_to_hit = L.get_bodypart(pick(BODY_ZONE_HEAD, BODY_ZONE_CHEST, BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG))
-			L.apply_damage(25, BRUTE, limb_to_hit, L.run_armor_check(limb_to_hit, "melee", null, null, armour_penetration))
-	sleep(3)
-
-/mob/living/simple_animal/hostile/megafauna/bubblegum/proc/bloodgrab(turf/T, handedness)
-	if(handedness)
-		new /obj/effect/temp_visual/bubblegum_hands/rightpaw(T)
-		new /obj/effect/temp_visual/bubblegum_hands/rightthumb(T)
-	else
-		new /obj/effect/temp_visual/bubblegum_hands/leftpaw(T)
-		new /obj/effect/temp_visual/bubblegum_hands/leftthumb(T)
-	sleep(6)
-	for(var/mob/living/L in T)
-		if(!faction_check_mob(L))
-			to_chat(L, "<span class='userdanger'>[src] drags you through the blood!</span>")
-			playsound(T, 'sound/magic/enter_blood.ogg', 100, 1, -1)
-			var/turf/targetturf = get_step(src, dir)
-			L.forceMove(targetturf)
-			playsound(targetturf, 'sound/magic/exit_blood.ogg', 100, 1, -1)
-			if(L.stat != CONSCIOUS)
-				addtimer(CALLBACK(src, .proc/devour, L), 2)
-	sleep(1)
-
-/obj/effect/temp_visual/dragon_swoop/bubblegum
-	duration = 10
-
-/obj/effect/temp_visual/bubblegum_hands
-	icon = 'icons/effects/bubblegum.dmi'
-	duration = 9
-
-/obj/effect/temp_visual/bubblegum_hands/rightthumb
-	icon_state = "rightthumbgrab"
-
-/obj/effect/temp_visual/bubblegum_hands/leftthumb
-	icon_state = "leftthumbgrab"
-
-/obj/effect/temp_visual/bubblegum_hands/rightpaw
-	icon_state = "rightpawgrab"
-	layer = BELOW_MOB_LAYER
-
-/obj/effect/temp_visual/bubblegum_hands/leftpaw
-	icon_state = "leftpawgrab"
-	layer = BELOW_MOB_LAYER
-
-/obj/effect/temp_visual/bubblegum_hands/rightsmack
-	icon_state = "rightsmack"
-
-/obj/effect/temp_visual/bubblegum_hands/leftsmack
-	icon_state = "leftsmack"
+	charging = 0
 
-/mob/living/simple_animal/hostile/megafauna/bubblegum/proc/blood_warp()
-	if(Adjacent(target))
-		return FALSE
-	var/list/can_jaunt = get_pools(get_turf(src), 1)
-	if(!can_jaunt.len)
-		return FALSE
-
-	var/list/pools = get_pools(get_turf(target), 2)
-	var/list/pools_to_remove = get_pools(get_turf(target), 1)
-	pools -= pools_to_remove
-	if(!pools.len)
-		return FALSE
-
-	var/obj/effect/temp_visual/decoy/DA = new /obj/effect/temp_visual/decoy(loc,src)
-	DA.color = "#FF0000"
-	var/oldtransform = DA.transform
-	DA.transform = matrix()*2
-	animate(DA, alpha = 255, color = initial(DA.color), transform = oldtransform, time = 3)
-	sleep(3)
-	qdel(DA)
 
+/mob/living/simple_animal/hostile/megafauna/bubblegum/proc/blood_warp()
 	var/obj/effect/decal/cleanable/blood/found_bloodpool
-	pools = get_pools(get_turf(target), 2)
-	pools_to_remove = get_pools(get_turf(target), 1)
```

Comment:
Seriously, this is literally the only change that matters in the entire PR:
It keeps bubblegum from porting directly to melee range, which means he can't port and immediately claw you.

Everything else is chaff. His charge actually gets slower and easier to react to.